backend/services/mgmt/v1alpha1/transformers-service/transformers_test.go

```diff
@@ -420,6 +420,43 @@ func Test_IsConnectionNameAvailable_False(t *testing.T) {
 	assert.Equal(t, false, resp.Msg.IsAvailable)
 }
 
+func Test_ValidateUserJavascriptCode_True(t *testing.T) {
+	m := createServiceMock(t)
+
+	code := `var payload = value+=" hello";return payload;`
```

Comment:
let's see an example where the script is actually on multiple lines

Reply:
updated